### ISSUE TITLE
Cursor position after inserting block embed

### ISSUE THREAD

**Issue by sdnnvs** [Roam-Research/issues#346]
The block embed code consists of {{ }} and (( )). Currently, after inserting the block embed, the cursor is parked at the end of )), so we have to go to the end of the code to finally hit Enter.

It would be best if, after inserting the block embed, the cursor was positioned after the }}, because, once the block embed was inserted, just press Enter to create a new bullet point.
With this, the process becomes more agile and natural.

Currently, in a hurry, when we insert the block embed and press Enter, the }} go together to the bullet point created on the line immediately below, which is frustrating.

In short, I opine that the cursor, after inserting the block embed, is positioned after the }} and not after the )), as it currently occurs.
